3rdparty/everest/include/everest/kremlib/FStar_UInt128.h

```diff
@@ -17,37 +17,48 @@
 #include <stdbool.h>
 #include "kremlin/internal/types.h"
 
-uint64_t FStar_UInt128___proj__Mkuint128__item__low(FStar_UInt128_uint128 projectee);
+uint64_t FStar_UInt128___proj__Mkuint128__item__low(
```

Comment:
Several files in everest were generated by Kremlin, and should be excluded from reformatting. Those are the files that currently don't have spaces inside parentheses.

Reply:
It looks like those files contain the useful string "This file was generated by KreMLin", so that should be easy enough to fix.

Reply:
As a general principle, shouldn't we not reformat 3rd party source (not just auto-generated 3rd party source)? Reformatting makes it harder to merge future upstream changes...

Reply:
Indeed we should avoid reformatting all the Kremlin files, both the ones that are generated by Kremlin and the ones (a couple of header files IIRC) that are support files distributed with Kremlin. So avoid reformatting files that already have a standard horizontal whitespace style, regardless of whether they're `generated by KreMLin`.

Reply:
I'm suggesting not automatically reformatting anything under `3rdparty/`

Reply:
Some of `3rdparty/everest` is Kremlin bundled/generated code, and some of it is glue code written specifically for Mbed TLS. The glue code was written in the Mbed TLS style and should be reformatted.

Reply:
`3rdparty` only contains everest and we currently have no plans to add anything else. So we don't need a general rule here.

The list of files we want to restyle should be the same list that's hard-coded in `check_names.py`, or close.

That being said,

> we don't apply formatting under 3rdparty, but as a one-off, we manually apply the script to the relevant files

seems like an attractive effort and runtime saving hack, given that the everest code practically never changes. We want the script to work on pending patches, but I think there are no pending patches to everest (at least none that have been submitted to us).